test/tcp/TcpServer.cpp

```diff
@@ -62,6 +62,11 @@ class ServerRequestHandler : public RequestHandler {
     response.onNext(folly::IOBuf::copyBuffer("from server2"));
     response.onComplete();
   }
+
+  void handleFireAndForgetRequest(Payload request) override {
+    std::cout << "ServerRequestHandler.handleFireAndForgetRequest "
```

Comment:
No good reason here.  This is short lived test code.

However if you compare to another client we expect users to use, it strictly uses STDOUT, STDERR such that unix pipes work. 

https://github.com/ReactiveSocket/reactivesocket-cli/blob/master/src/main/java/io/reactivesocket/cli/ConsoleOutputHandler.java